content/authors/authors.ts

```diff
@@ -117,7 +117,7 @@ export const Authors: Author[] = [
     photo: 'andi',
     name: 'Andi Qu',
     title: 'Co-Founder',
-    blurb: 'Andi Qu is an IOI and IMO bronze medalist.',
+    blurb: 'Andi Qu is an IOI and IMO bronze medalist, and part of the MIT Class of 2025',
```

Comment:
is this needed

Reply:
benq has it, I don't see why andi cant...

Reply:
I mean, technically it's appealing and attracts young coders.

If not we can remove it and focus on the small " error 🤷🏼‍♂️

Reply:
doesnt hurt to have it no cap
it creates the imagery "cp can get me into mit!??!?!!? no way allow me to grind cp from now on"

Reply:
wwait isn't up to andi if he wants to include it?